File: go/wam/6034.go
const WamCallInfoUserJourney = 6034 // channel: regular

type CallInfoUserJourneyEvent struct {
	AppSessionId string `wam:"app_session_id"` // field 1
	CallGroupSizeBucket CALLSIZEBUCKET `wam:"call_group_size_bucket"` // field 2
	CallSizeType CALLSIZETYPE `wam:"call_size_type"` // field 3
	CallType CALLTYPE `wam:"call_type"` // field 4
	PreCallActionType PRECALLACTIONTYPE `wam:"pre_call_action_type"` // field 5
	SurfaceSessionId string `wam:"surface_session_id"` // field 6
	UserJourneyEventMs int64 `wam:"user_journey_event_ms"` // field 7
	UserJourneyFunnelId string `wam:"user_journey_funnel_id"` // field 8
	NumParticipantsShown int64 `wam:"num_participants_shown"` // field 9
	ParticipantActionSource PARTICIPANTACTIONSOURCE `wam:"participant_action_source"` // field 10
	GenaiBots string `wam:"genai_bots"` // field 11
}

File: rust/wam/6034.rs
pub const WAM_CALL_INFO_USER_JOURNEY: u32 = 6034;

#[derive(Debug, Default)]
pub struct CallInfoUserJourneyEvent {
    /// field 1, wire `app_session_id`
    pub app_session_id: Option<String>,
    /// field 2, wire `call_group_size_bucket`
    pub call_group_size_bucket: Option<CALLSIZEBUCKET>,
    /// field 3, wire `call_size_type`
    pub call_size_type: Option<CALLSIZETYPE>,
    /// field 4, wire `call_type`
    pub call_type: Option<CALLTYPE>,
    /// field 5, wire `pre_call_action_type`
    pub pre_call_action_type: Option<PRECALLACTIONTYPE>,
    /// field 6, wire `surface_session_id`
    pub surface_session_id: Option<String>,
    /// field 7, wire `user_journey_event_ms`
    pub user_journey_event_ms: Option<i64>,
    /// field 8, wire `user_journey_funnel_id`
    pub user_journey_funnel_id: Option<String>,
    /// field 9, wire `num_participants_shown`
    pub num_participants_shown: Option<i64>,
    /// field 10, wire `participant_action_source`
    pub participant_action_source: Option<PARTICIPANTACTIONSOURCE>,
    /// field 11, wire `genai_bots`
    pub genai_bots: Option<String>,
}